mkdirp/.travis.yml

```diff
@@ -0,0 +1,7 @@
+os:
```

Comment:
I think we can skip this file, CI is already set in root directory

Reply:
Yes, please remove these CI files.